### PR TITLE
add docs to get_view.dart

### PR DESCRIPTION
**Pull request by roipeker**
very simple doc reference.

### CODE REVIEW

**Comment by jonataslaw** [comment]
LGTM! Thanks

File: lib/src/state_manager/simple/get_view.dart [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/widgets.dart';
 import 'package:get/src/instance/get_instance.dart';
 
+/// GetView is a great way of quickly access your Controller
+/// without having call Get.find<Controller>() yourself.
```
Comment (ghprod) [inline lib/src/state_manager/simple/get_view.dart:5]: @roipeker maybe you can add how to use it, something like this.

```dart
class SomeScreen extends StatelessWidget {}

// change to

class SomeScreen extends GetView<SomeController> {}
```

Comment (roipeker) [reply lib/src/state_manager/simple/get_view.dart:5]: Sure!, it was a quick test i was doing with @Nipodemos with the PRs yesterday.

**Comment by jonataslaw** [comment]
LGTM

**Pull request merged**